lldb/source/Plugins/LanguageRuntime/Swift/SwiftTask.h

```diff
@@ -0,0 +1,93 @@
+
+#include "lldb/Target/RegisterContext.h"
+#include "lldb/Target/Thread.h"
+#include "lldb/Utility/RegisterValue.h"
+#include "lldb/lldb-forward.h"
+
+namespace lldb_private {
+
+using namespace lldb;
+
+/// Provides a subset of Thread operations for Swift Tasks.
+///
+/// Currently, this supports backtraces of Tasks, and selecting frames in the
+/// backtrace. Async frames make available the variables that are stored in the
+/// Task's "async context" (instead of the stack).
+///
+/// See `Task<Success, Failure>` and `UnsafeCurrentTask`
+class ThreadTask : public Thread {
```

Comment:
I haven't been able to decide if we should say `SwiftTask` instead of just `Task` to make it plainly obvious that all of these are Swift constructs. Any thoughts?

Reply:
both `ThreadTask` and `ThreadSwiftTask` work for me.